tests/e2e/actor_state/actor_state_test.go

```diff
@@ -0,0 +1,308 @@
+//go:build e2e
```

Comment:
I would like this to be a standalone test since we are solely focused on state, _and_ using both grpc and http.

As a separate effort, I would like to see the apps consolidated into a one or few numbers of apps/images.